rubocop-migrations.gemspec

```diff
@@ -4,32 +4,25 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'rubocop/migrations/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "rubocop-migrations"
+  spec.name          = 'rubocop-migrations'
   spec.version       = Rubocop::Migrations::VERSION
-  spec.authors       = ["enricostano"]
-  spec.email         = ["ocirne.onats@gmail.com"]
 
-  spec.summary       = ""
-  spec.description   = ""
-  spec.homepage      = ""
-  spec.license       = "MIT"
+  spec.authors       = %w(enricostano magec)
+  spec.email         = %w(ocirne.onats@gmail.com joseferper@gmail.com)
 
-  # Prevent pushing this gem to RubyGems.org by setting 'allowed_push_host', or
-  # delete this section to allow pushing this gem to any host.
-  if spec.respond_to?(:metadata)
-    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
-  else
-    raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
-  end
+  spec.summary       = 'Rails migrations rubocop checks'
+  spec.description   = 'Performs some checks for migration sanity'
+  spec.homepage      = 'https://github.com/redbooth/rubocop-migrations'
+  spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) } # rubocop:disable Metrics/LineLength
+  spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  spec.require_paths = ['lib']
 
   spec.add_runtime_dependency 'rubocop', '>= 0.42.0'
 
-  spec.add_development_dependency "bundler", "~> 1.11"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "3.5.0"
+  spec.add_development_dependency 'bundler', '~> 1.11'
```

Comment:
I believe the latest is 1.13 something. I would relax it more to be able to use the latest.